.github/release-drafter-engine.yml

```diff
@@ -1,11 +0,0 @@
-_extends: kaskada:.github/release-drafter.yml
```

Comment:
We can probably combine the general `release-drafter.yml` and the `release-drafter-python.yml` since we don't need the "sharing".